libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/stats/stats.json

```diff
@@ -1,7 +1,15 @@
 {
     "request": {
         "method": "GET",
-        "urlPattern": "/rest/v1.1/sites/158396482/stats(/+)?($|\\?.*)"
+        "urlPath": "/rest/v1.1/sites/106707880/stats//",
```

Comment:
There's an extra `/` at the end of the path here, causing a mocks failure.

Reply:
I was looking at this one trying to remember what happened, because I thought there was a good reason – turned out there was – iOS requires the `//`, or it fails to match the request.

IMHO – this is similar to the issue with the random image IDs and we should leave it with a single slash as it:

A) Doesn't break the tests on iOS (whereas it _does_ break things on Android)
B) Is a bug in WPiOS that should probably be fixed

I tried using the urlPattern instead, but despite trying several different ways, WireMock is refusing to match that URL for whatever reason :(

Reply:
I figured it out – over-using `urlPattern` in stats broke this, so I've replaced all instances of it there with this, and it now works correctly and fixes this issue.

Reply:
(FWIW I still think we should fix the bug in iOS that causes this behaviour)

Reply:
> iOS requires the //, or it fails to match the request.

o.O Agreed that the bug should be fixed. Could you open an issue on the iOS side for that bug?